buggy-queries.txt

```diff
@@ -0,0 +1,2 @@
+what is the average salary of employees where name is not vinh
+how many jobs where name is not vinh
```

Comment:
I ran it on the old version and it didnt null pointer, I know that average isnt a word in the old code but its possible you broke something?